tests/foreman/api/test_role.py

```diff
@@ -292,8 +489,15 @@ def test_positive_create_org_admin_from_clone(self):
 
         :caseautomation: notautomated
         """
+        default_org_admin = entities.Role().search(
+            query={'search': u'name="Organization admin"'})
+        org_admin = self.create_org_admin()
+        default_filters = entities.Role(
+            id=default_org_admin[0].id).read_json()['filters']
```

Comment:
Pls avoid using `.read_json()` where it's possible. If `filters` fields is not present in `entities.Role()` - then pls add it and use regular `.read()`